templates/oauth_result.html.njk

```diff
@@ -0,0 +1,68 @@
+        <h1>Slack Integration {{"Successful" if success else "Failed"}} </h1>
+        <h2><code>{{ code }}</code></h2>
+    </section>
+    <section>
+        {% if success %}
+        <p>Your account (<code>{{ userId }}</code>) has been successfully connected to the Slack bridge.</p>
+        <small>You can safely close this window</small>
+        {% else %}
+            {% if reason === "token-not-known" %}
+                <p>The bridge cannot handle this request because the token provided is not known.</p>
+                <small>This can happen if the bridge was recently restarted. Try running the oauth process again.</small>
+            {% elif reason === "limit-reached" %}
+                <p>You have reached the maximum number of connected teams on this bridge instance.</p>
+                <small>Please contact your bridge administrator</small>
```

Comment:
I don't expect people know who their bridge admin is.
I think we should have a config field for the admin to leave their free text contact, be it a name, a Matrix ID or whatever.

Reply:
I'm really on the fence about it. It feels like we could just tell admins to add their name to the template (as I think matrix.org would definitely add some custom content to this). 

Otoh, maybe customizing templates is more than most admins want to do.

Reply:
good point, they can modify the templates.